packages/types/src/accountability.ts

```diff
@@ -9,9 +9,9 @@ export type Accountability = {
 	user: string | null;
 	admin: boolean;
 	app: boolean;
-	share?: string;
+	share?: string | undefined;
 	ip: string | null;
-	userAgent?: string;
-	origin?: string;
-	session?: string;
+	userAgent?: string | undefined;
+	origin?: string | undefined;
+	session?: string | undefined;
```

Comment:
Not really sure about editing `Accountability` just for convenience, as it is used everywhere. And the expectation could have been in some block of logic, that if the key is set, that there also is a value present in accountability.

Reply:
Fair point, reverted in 8983dcebb4574ef95a3ae8a4b83309a43e026cce